src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got /tmp/tmp3x8L4M.tmp/testDllNativeFolder/linux-x64;/tmp/tmp3x8L4M.tmp/testDllNativeFolder/managed.dll.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 64
64:         (dir $root)  -join ";" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Unable to find type [TestNativeClass].
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 76
76:         [TestNativeClass]::LoadNative() | Should -Not -BeNullOrEmpty

```